buildpipeline/pipelines.json

```diff
@@ -352,6 +391,11 @@
             "GitHubRepositoryName": "coreclr",
             "AzureContainerPackageGlob": "pkg\\*.nupkg",
             "AzureContainerSymbolPackageGlob": "symbolpkg\\*.nupkg"
+          },
+          "ReportingParameters": {
+            "SubType":  "Publish",
+            "Type": "build/publish/",
+            "PB_BuildType": "Release"
```

Comment:
I'm not certain if the viewconfiguration file for MC needs to be updated to recognize PB_BuildType or if that already happened for CoreClr.